libparsec/crates/client/src/monitors/connection.rs

```diff
@@ -157,6 +157,7 @@ fn dispatch_api_event(event: APIEvent, event_bus: &EventBus) {
             };
             event_bus.send(&event);
         }
+        APIEvent::ChatMessageReceived { .. } => (),
```

Comment:
```suggestion
        APIEvent::ChatMessageReceived { .. } => todo!("Chat message received"),
```

Put a `todo!` comment here, this will make the code panic here